src/docfx/Models/MergeCommandOptions.cs

```diff
@@ -40,5 +40,8 @@ internal class MergeCommandOptions : ICanPrintHelpMessage, ILoggable
 
         [Option("fileMetadataFile", HelpText = "Specify a JSON file path containing fileMetadata settings, as similar to {\"fileMetadata\":{\"key\":\"value\"}}. It overrides the fileMetadata settings from the config file.")]
         public string FileMetadataFilePath { get; set; }
+
+        [OptionList("metadataNeedMergedIntoToc", Separator = ',', HelpText = "Specify metadata names that need to be merged into toc file")]
```

Comment:
how about `tocMetadata`... it's still a little weird...

---
In reply to: [90402659](https://github.com/dotnet/docfx/pull/941#discussion_r90402659) [](ancestors = 90402659)